packages/desktop-client/src/components/reports/graphs/tableGraph/ReportTableRow.tsx

```diff
@@ -77,183 +86,195 @@ export const ReportTableRow = memo(
-                />
-                <Cell
-                  value={amountToCurrency(item.totalDebts)}
-                  title={
-                    Math.abs(item.totalDebts) > 100000
-                      ? amountToCurrency(item.totalDebts)
-                      : undefined
-                  }
-                  width="flex"
-                  privacyFilter
-                  style={{
-                    minWidth: compact ? 50 : 85,
-                  }}
-                  linkStyle={hoverUnderline}
-                  valueStyle={compactStyle}
-                  onClick={() =>
-                    !isNarrowWidth &&
-                    !['Group', 'Interval'].includes(groupBy) &&
-                    !categories.grouped.map(g => g.id).includes(item.id) &&
-                    showActivity({
-                      navigate,
-                      categories,
-                      accounts,
-                      balanceTypeOp,
-                      filters,
-                      showHiddenCategories,
-                      showOffBudget,
-                      type: 'debts',
-                      startDate,
-                      endDate,
-                      field: groupBy.toLowerCase(),
-                      id: item.id,
-                    })
-                  }
-                />
-              </>
-            )}
-        <Cell
-          value={amountToCurrency(item[balanceTypeOp])}
-          title={
-            Math.abs(item[balanceTypeOp]) > 100000
-              ? amountToCurrency(item[balanceTypeOp])
-              : undefined
-          }
-          style={{
-            fontWeight: 600,
-            minWidth: compact ? 50 : 85,
-          }}
-          linkStyle={hoverUnderline}
-          valueStyle={compactStyle}
-          onClick={() =>
-            !isNarrowWidth &&
-            !['Group', 'Interval'].includes(groupBy) &&
-            !categories.grouped.map(g => g.id).includes(item.id) &&
-            showActivity({
-              navigate,
-              categories,
-              accounts,
-              balanceTypeOp,
-              filters,
-              showHiddenCategories,
-              showOffBudget,
-              type: 'totals',
-              startDate,
-              endDate,
-              field: groupBy.toLowerCase(),
-              id: item.id,
-            })
-          }
-          width="flex"
-          privacyFilter
-        />
-        <Cell
-          value={integerToCurrency(Math.round(average))}
-          title={
-            Math.abs(Math.round(average / 100)) > 100000
-              ? integerToCurrency(Math.round(average))
-              : undefined
-          }
-          style={{
-            fontWeight: 600,
-            minWidth: compact ? 50 : 85,
-          }}
-          valueStyle={compactStyle}
-          width="flex"
-          privacyFilter
-        />
+        >
+          <Cell
+            value={item[groupByItem]}
+            title={item[groupByItem] || undefined}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this necessary?
```suggestion
            title={item[groupByItem]}
```